spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+feature 'comments' do
+  let!(:pfc) {Picture.create(title:'PFC',
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Space missing after colon.